app/components/lr-progress.js

```diff
@@ -0,0 +1,55 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  classNames: ['lr-progress'],
+
+  classNameBindings: ['isCompleted:lr-progress--is-completed', 'success:lr-progress--success'],
+
+  reached: 0,
+
+  total: 100,
+
+  withAchievementPoints: false,
+
+  percentage: Ember.computed('reached', 'total', function () {
+
+    return this.get('reached') / this.get('total') * 100;
+
+  }),
+
+  isCompleted: Ember.computed('percentage', function () {
+
+    if (Number(this.get('percentage')) === 100) {
+
+      return true;
+
+    } else {
+
+      return false;
+
+    }
+
+  }),
+
+  inProgress: Ember.computed('percentage', function () {
+
+    if (Number(this.get('percentage')) < 100 && Number(this.get('percentage')) > 0) {
```

Comment:
The reason I implemented it this way was because I wanted to show 3 different styles for
- 0% - not began
- inProgress for everything between 0% & 100%
- isCompleted
But if you can live with only the two styles for 0-99% and isCompleted then I'll remove the inProgress property and styles for inProgress will become default styles 🙊